api_app/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.23.0"
+__version__ = "0.23.1"
```

Comment:
Remember to bump this to 0.24 before merge.

Reply:
```suggestion
__version__ = "0.24.0"
```

Reply:
Done